NuEVI/NuEVI.ino

```diff
@@ -261,6 +258,16 @@ bool configManagementMode = false;
 
 //_______________________________________________________________________________________________ SETUP
 
+//Update CV output pin, run from timer.
+void cvUpdate(){
+  int cvPressure = analogRead(breathSensorPin);
+  if(dacMode == DAC_MODE_PITCH){
+    analogWrite(pwmDacPin,cvPressure);
+  } else { //DAC_MODE_BREATH
+    analogWrite(dacPin,map(constrain(cvPressure,breathThrVal,4095),breathThrVal,4095,0,4095));
+  }
+}
+
```

Comment:
I moved this up here so the simulator would compile. Arduino does some hidden magic to allow functions to be called even if they're only defined "later", but this makes it work with normal C++ too.